packages/@okta/i18n/src/properties/login.properties

```diff
@@ -699,15 +699,15 @@ cert.authentication.title = Certificate authentication
 ##################################################
 
 ## Password
-oie.password = Password
+oie.password.authenticator.description = Choose a password for your account
```

Comment:
The changes here is to make the labels more specific in compliance with i18nTransformer labels.